Wireframe/css/style.css

```diff
@@ -0,0 +1,73 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  text-decoration: none;
+}
+body {
+  background-color: #333333;
+}
+header {
+  text-align: center;
+  padding: 20px;
+  background-color: #f4f4f4;
+}
+h1 {
+  /* margin-bottom: 30px; */
+  padding: 30px;
+}
+main {
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  grid-template-rows: auto auto;
+  gap: 20px;
+  padding: 20px;
+  max-width: 1200px;
+  margin: 0 auto;
+}
+
+.article {
+  background-color: #f4f4f4;
+  border: 1px solid #ddd;
+  padding: 20px;
+  display: flex;
+  flex-direction: column;
+}
+.article:first-child {
+  grid-column: 1 / -1;
+}
+
+.article-photo {
+  width: 100%;
+  aspect-ratio: 3/1;
```

Comment:
Aspect-ratio made photos a little odd. There is one css style you can add - `object-fit`. Try it out, e.g. `object-fit: cover`. This will fit the pictures into its container. You can play with the property to find better solution.